pkg/features/kube_features.go

```diff
@@ -252,6 +258,9 @@ var defaultVersionedFeatureGates = map[featuregate.Feature]featuregate.Versioned
 	TASProfileMixed: {
 		{Version: version.MustParse("0.11"), Default: false, PreRelease: featuregate.Deprecated},
 	},
+	HierarchialCohorts: {
+		{Version: version.MustParse("0.12"), Default: true, PreRelease: featuregate.Beta},
```

Comment:
I think if you want to cherry pick this may require manual adjustment to the PR to set to 11.